src/pipecat/serializers/exotel.py

```diff
@@ -0,0 +1,162 @@
+        self._sample_rate = self._params.sample_rate or frame.audio_in_sample_rate
+
+    async def serialize(self, frame: Frame) -> str | bytes | None:
+        """Serializes a Pipecat frame to Exotel WebSocket format.
+
+        Handles conversion of various frame types to Exotel WebSocket messages.
+
+        Args:
+            frame: The Pipecat frame to serialize.
+
+        Returns:
+            Serialized data as string or bytes, or None if the frame isn't handled.
+        """
+        if isinstance(frame, StartInterruptionFrame):
+            answer = {"event": "clear", "streamSid": self._stream_sid}
+            return json.dumps(answer)
+        elif isinstance(frame, AudioRawFrame):
+            data = frame.audio
+
+            # Resample the audio data
+            serialized_data = await self._resampler.resample(
+                data, frame.sample_rate, self._params.exotel_sample_rate
+            )
+            payload = base64.b64encode(serialized_data).decode("ascii")
+
+            answer = {
+                "event": "media",
+                "streamSid": self._stream_sid,
+                "media": {"payload": payload},
+            }
+
+            return json.dumps(answer)
+        elif isinstance(frame, (TransportMessageFrame, TransportMessageUrgentFrame)):
+            return json.dumps(frame.message)
+
+        return None
+
+    async def deserialize(self, data: str | bytes) -> Frame | None:
+        """Deserializes Exotel WebSocket data to Pipecat frames.
+
+        Handles conversion of Exotel media events to appropriate Pipecat frames.
+
+        Args:
+            data: The raw WebSocket data from Exotel.
+
+        Returns:
+            A Pipecat frame corresponding to the Exotel event, or None if unhandled.
+        """
+        message = json.loads(data)
+
+        if message["event"] == "media":
+            payload_base64 = message["media"]["payload"]
+            payload = base64.b64decode(payload_base64)
+
+            # Input: Convert Exotel's 8kHz μ-law to PCM at pipeline input rate
+        if message["event"] == "media":
+            payload_base64 = message["media"]["payload"]
+            payload = base64.b64decode(payload_base64)
+
+            deserialized_data = await self._resampler.resample(
+                payload,
+                self._params.exotel_sample_rate,
+                self._params.sample_rate,
+            )
+
+            audio_frame = InputAudioRawFrame(
+                audio=deserialized_data,
+                num_channels=1,
+                sample_rate=self._params.sample_rate,
+            )
+            return audio_frame
+        elif message["event"] == "dtmf":
+            digit = message.get("dtmf", {}).get("digit")
+
+            try:
+                return InputDTMFFrame(KeypadEntry(digit))
+            except ValueError:
+                # Handle case where string doesn't match any enum value
+                loguru.logger.info(f"Invalid DTMF digit: {digit}")
```

Comment:
Aligning with how logger is used elsewhere.

```suggestion
                logger.info(f"Invalid DTMF digit: {digit}")
```